app/Shell/DockerTags.php

```diff
@@ -44,7 +44,12 @@ public function getTags(): Collection
     {
         $response = json_decode($this->getTagsResponse()->getContents(), true);
 
-        return collect($response['results'])->map->name->filter();
+        return collect($response['results'])
+            ->map(function ($item) {
```

Comment:
OK, so in this one you're turning it into something like this, right?

```
[
    ['name' => '8.4.0'],
    ['name' => '8.3.1'],
]
```

Could pluck the name first and then sort? I don't have the `sortDesc` code in front of me, so I don't know if it takes the sort type like sortbydesc does... but something like this?

```php
return collect($response['results'])
    ->pluck('name')
    ->sortDesc(SORT_NATURAL)
    ->filter();
```

Reply:
done